pkg/client/client_test.go

```diff
@@ -707,7 +711,8 @@ var _ = Describe("JSONLogger", func() {
 						Path: tr.logPath(),
 					}},
 				})
-				Expect(err).To(HaveOccurred())
+				Expect(err).NotTo(HaveOccurred())
+				tr.startContainer(sut)
```

Comment:
it's not clear to me why this switched

Reply:
I changed the container above to actually be valid. Fun fact is that the for loop `for _, terminal := range []bool{true, false} {` in go 1.21 (ginkgo related?) does not work for `true`, but it works in go 1.22.